engines/python/setup/djl_python/tensorrt_llm.py

```diff
@@ -29,6 +30,68 @@
 from djl_python.properties_manager.properties import is_rolling_batch_enabled
 
 
+def _get_value_based_on_tensor(value, index=None):
```

Comment:
could we fuse all python backend interaction into a single class (file)? Just trying to keep the handler clean

Reply:
Sure, we could do that. Same as stable diffusion handlers.

Reply:
Sure, we could do that. Same as stable diffusion handlers.